File: demo_oracle_db/src/main/java/com/example/demo_oracle_db/service/mySupplier/response/SupplierInfo.java
package com.example.demo_oracle_db.service.mySupplier.response;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
public class SupplierInfo {
    private Long id;
    private String name;
    private String contact;
    private String address;
    private String phone;
    private String email;
    private String website;
}

File: demo_angular/src/app/model/supplier/supplier-info.ts
export interface SupplierInfor{
  id: number
  name: string
  contact: string
  address: string
  phone: string
  email: string
  website: string
}

export interface SupplierFilter{
  name: string|null
  contact: string|null
  address: string|null
  phone: string|null
  email: string|null
  website: string|null
  pageNum: number;
  pageSize: number;
}
